src/base/bittorrent/tracker.cpp

```diff
@@ -271,51 +274,93 @@ void Tracker::processAnnounceRequest()
     // ip address
     announceReq.socketAddress = m_env.clientAddress;
     announceReq.claimedAddress = queryParams.value(ANNOUNCE_REQUEST_IP);
+    // request port
+    const quint16 requestPort =  m_env.clientPort;
 
     // Enforce using IPv4 if address is indeed IPv4 or if it is an IPv4-mapped IPv6 address
     bool ok = false;
     const qint32 decimalIPv4 = announceReq.socketAddress.toIPv4Address(&ok);
-    if (ok)
+    if (ok) {
         announceReq.socketAddress = QHostAddress(decimalIPv4);
+        LogMsg(tr("Embedded Tracker: Announce request from non- plain-IPv6 address %1."
```

Comment:
Due to comment above this message is incorrect to be here since this place can be reached in case of plain IPv4 address.

Reply:
@glassez 
This message is supposed to be printed every time anything _except_ a **plain IPv6** is used. This includes the case of a **plain IPv4**.
In other words, this message prints "if address is indeed IPv4 or if it is an IPv4-mapped IPv6 address", which is what the comment above says.

The point of this is to help track and diagnose connectivity issues with the embedded tracker when IPv4-mapped IPv6 addresses are used and possibly other cases.

Reply:
>This includes the case of a plain IPv4.

But it is a trivial case. Why should it be logged (unlike plain IPv6)?

Reply:
@glassez because I did not find an easy way to exclude it.

Reply:
Can't you check whether it was IPv4 in the first place (before it was adjusted)?

Reply:
@glassez Of course I can. Brain freeze, sorry. It's fixed now.